File: 27RemoveElement/RemoveElement.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int validSpace = 0;
        for(int i = 0; i < nums.Length; i++){
            if(nums[i] != val){
                nums[validSpace++] = nums[i];
            }
        }
        return validSpace;
    }
}

File: 27RemoveElement/removeElement.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int validIndex = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                nums[validIndex++] = nums[i];
            }
        }

        return validIndex;
    }
}
